packages/mcp/src/microsoft/teams/mcp/ai_plugin.py

```diff
@@ -0,0 +1,262 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+import asyncio
+import json
+import logging
+import time
+from typing import Any, Awaitable, Callable, Dict, List, Optional, Tuple, Union
+
+from microsoft.teams.ai.function import Function
+from microsoft.teams.ai.plugin import BaseAIPlugin
+from microsoft.teams.common.logging import ConsoleLogger
+from pydantic import BaseModel
+
+from mcp import ClientSession
```

Comment:
nit: move external imports to top (mb we can add this into ruff?)

Reply:
Let me do this in a separate PR. I'll run import reordering across all files

Reply:
https://docs.astral.sh/ruff/settings/#lint_isort_sections going to use this (just so i remember)